### ISSUE TITLE
How to remove the ConfirmationCallback in fr-oath module

### ISSUE THREAD

**Issue by lijinlongpq** [WrenSecurity/wrenam#147]
Hi, 

I am setting up my VPN server with 2FA using the ForgeRock Authenticator (OATH) module with wrenam 15.0.0-M6. I already registered my device and set Two Factor Authentication Mandatory. 
I can login from web browser with OTP, no problem. 

But when I login from VPN client, I prompt me 2 times for second password. 

1. > **ForgeRock Authenticator (OATH) Enter verification code**
2. > **ForgeRock Authetnicator (OATH) (Default is 0) 0=Submit**

Then I tried from Postman. The response for second time password contains 2 call backs, NameCallback & ConfirmationCallback. Seems this is causing the problem. I think the ConfirmationCallback is just a Submit button. Is there a way to remove the ConfirmationCallback. Like the DataStore module, just return a NameCallback and a PasswordCallback.
{
    "authId": "eyJ0eXAi...",
    "template": "",
    "stage": "AuthenticatorOATH4",
    "header": "ForgeRock Authenticator (OATH)",
    "callbacks": [
        {
            "type": "NameCallback",
            "output": [
                {
                    "name": "prompt",
                    "value": "Enter verification code:"
                }
            ],
            "input": [
                {
                    "name": "IDToken1",
                    "value": "jinlong"
                }
            ]
        },
        {
            "type": "ConfirmationCallback",
            "output": [
                {
                    "name": "prompt",
                    "value": ""
                },
                {
                    "name": "messageType",
                    "value": 0
                },
                {
                    "name": "options",
                    "value": [
                        "Submit"
                    ]
                },
                {
                    "name": "optionType",
                    "value": -1
                },
                {
                    "name": "defaultOption",
                    "value": 0
                }
            ],
            "input": [
                {
                    "name": "IDToken2",
                    "value": 0
                }
            ]
        }
    ]
}

**Comment by pavelhoral** [comment]
You need to send the same number of callbacks as the server requests. It is possible to remove the callback from the auth module definition XML file, but I would not go that way. Just check what the XUI sends to `/authenticate` endpoint when you login through browser.

**Comment by lijinlongpq** [comment]
Thanks, 

I tried remove the confirmationcallback from the XML file. And I can login to my VPN server using radius with just OTP, it's what I wanted flow.
After remove the confirmationcallback , using REST API login will not work, and I think it's expected. 

I tried to build my custom authentication module, but it's not shown in the console. I cannot add the custom module instance. 
Is there any guide for me to install/register my custom authentication module ?
